src/model/scripts/finetune_llama2/OpenThaiGPT_FineTune_Llama.ipynb

```diff
@@ -0,0 +1,1474 @@
+{
```

Comment:
Can you put this into folder `src/model/notebook/finetune_llama2`. If the folder did not exist, please create it.

Reply:
done